### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by xzenor** [saltstack/salt-winrepo-ng#1899]
The salt-winrepo got updated saying everyone should go to salt-winrepo-ng.. but this seems to be just as dead.

**Comment by H20-17** [comment]
I've got two 26 day old pull requests. In the past, it never too a long time to get them merged.

**Comment by xzenor** [comment]
So.. you're not sure either?

**Comment by H20-17** [comment]
@xzenor  I'm worried. I wish there would be some transparency about what's going on.

**Comment by xzenor** [comment]
@H20-17 I share your worries... :(

**Comment by H20-17** [comment]
@twangboy, i hate to bug you. I know that you do massive amounts of work for salt in general, and I have no idea what your actual obligations are (if you've got any at all). Can you at least tell us what's happening?

**Comment by H20-17** [comment]
@xzenor my pr's were accepted the other day, so I guess things are still alive.

**Comment by xzenor** [comment]
Yeah I've gotten some comments on mine as well.

**Comment by twangboy** [comment]
Yeah, I'm the only one who monitors this and sometimes I get a little busy and don't check for pending PRs. Sorry about that. I'll try to keep a better eye on that.

**Comment by xzenor** [comment]
No worries. Life is important as well.
Would you prefer to be mentioned whenever a pull request is waiting so you get notified?

**Comment by ricekab** [comment]
It might be worth appointing some additional maintainers to take the load off. Are there formal processes for this?

**Comment by xzenor** [comment]
It's important to get at least one extra maintainer just in case something happens to that 1 maintainer. It's also convenient for faster pr acceptance.

**Comment by xzenor** [comment]
@twangboy Could you check the pending PR's again please?

**Comment by twangboy** [comment]
I've been trying to monitor this repo at least once a week.
